README.md

```diff
@@ -1,2 +1,43 @@
-# dry21-scripts
-Enapter Dryer DRY 2.1 control scripts examples 
+# Enapter Dryer DRY 2.1 control scripts examples 
+
+Find out more about [Enapter Dryer DRY 2.1](https://handbook.enapter.com/electrolyser/dry21/dry21.html).
+
+---
+
+Example scripts need pymodbus package to be installed. You can install it running: 
+`pip3 install pymodbus`
+
+
+
+## dry21_modbus_control.py
+#### Your devices (Enapter Electrolyser(s) EL 2.1 and Dryer DRY 2.1) must be connected to either [Dryer Control Network](https://handbook.enapter.com/electrolyser/el21a/el21a.html#dryer-control-network-with-enapter-cloud-connection) or [Isolated Dryer Control Network](https://handbook.enapter.com/electrolyser/el21a/el21a.html#isolated-dryer-control-network). 
+
+This script allows to control Enapter Dryer DRY 2.1 over Modbus TCP. Use it if you want Dryer to:
```

Comment:
This script allows controlling Enapter Dryer  DRY 2.1 over Modbus TCP of Electrolyser unified with DCN/iDCN